verifier/src/evidence.rs

```diff
@@ -245,7 +245,7 @@ mod test {
                   - [x] The ISV product ID should be 0
                   - [x] The ISV SVN should be at least 0
                 - [x] Both of the following must be true:
-                  - [x] The expected attributes is Flags: INITTED | DEBUG | MODE_64BIT Xfrm: LEGACY | AVX with mask Flags: 0xFFFF_FFFF_FFFF_FFFF Xfrm: LEGACY | AVX | AVX_512 | MPX | PKRU | AMX | RESERVED
+                  - [x] The expected attributes is Flags: INITTED | DEBUG | MODE_64BIT Xfrm: LEGACY | AVX with mask Flags: 0xFFFF_FFFF_FFFF_FFFF Xfrm: LEGACY | AVX | AVX_512 | MPX | PKRU | RESERVED
```

Comment:
Note how the `AMX` value that was shadowing the `LEGACY` value is no longer displayed in the output.
While it was an error for `AMX` to shadow `LEGACY`, https://github.com/mobilecoinfoundation/sgx/pull/354, this still could have happened if there was an actual shadow or alias bitflag.